src/openrct2/network/Network.cpp

```diff
@@ -1502,6 +1506,9 @@ void Network::EnqueueGameAction(const GameAction *action)
 
 void Network::AddClient(ITcpSocket * socket)
 {
+    if (gConfigNetwork.pause_server && game_is_paused()) {
```

Comment:
Could you put the braces in their own lines (code style)?

Reply:
I've looked around the other style within this file and adopted this. For sure I will fix this.

Reply:
Yeah, many files do not adhere to it yet, unfortunately.

Reply:
I've not found a strict rule within contribution guideline for braces. What's about single line `if` statement it's allowed to drop the curly braces?